assets/src/styles/blocks/Covers/ContentCovers.scss

```diff
@@ -81,16 +81,17 @@
         text-decoration: underline;
       }
 
-      a {
-        color: $blue;
-      }
-
       h4 {
         font-size: $font-size-xxs;
         line-height: 1.2;
         font-weight: 500;
         padding: 0;
         margin: 0;
+        color: $grey-80;
```

Comment:
In the Articles block the same is achieved by using inherit. Maybe makes sense here too, so we don't have to specify the text color again here? Also noticed that for articles we use `<h4><a>...</a></h4>`, while covers uses `<a><h4>...</h4></a>` :thinking: 
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/e5519762ac8d0d3f8759ed2de834e033f06ee66e/assets/src/styles/blocks/Articles.scss#L27-L27

Reply:
`color: inherit;` doesn't work, it just applies the link color (also when inverting the tags)... In the Articles block it's not what changes the heading color, it's [this line](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/styles/blocks/Articles.scss#L91) so it looks like we do need to specify the color again :/

Reply:
However for Columns block we do also use this order `<h3><a>...</a></h3>` and then in the css we use the selector `h3 > a` to change the color, I think that should work here too and would add more consistency in the code 🤔

Reply:
Just pushed a comment with those changes, please have a look @Inwerpsel 🙂 It also allows us to get rid of adding the hover styles to `h4`

Reply:
Yeah there is in fact a significant difference between having the heading on the inside or outside of the link. If it's on the inside then the clickable area will span the whole width, while on the outside the clickable area will be only the text. So having the heading on the outside seems preferable in this case, to avoid unintentional clicks of the link :+1:

Reply:
https://css-tricks.com/link-header-header-link/